### ISSUE TITLE
mock.patch.multiple(..., **patches) type checks against non-**kwargs arguments

### ISSUE THREAD

**Issue by asottile** [python/mypy#8525]
example:

```python
import os
from unittest import mock


patches = {'listdir': lambda x: ['t']}
with mock.patch.multiple(os, **patches):
    pass
```

output:

```console
$ mypy --version
mypy 0.770
$ mypy t.py
t.py:6: error: Argument 2 to "multiple" of "_patcher" has incompatible type "**Dict[str, Callable[[Any], List[str]]]"; expected "bool"
Found 1 error in 1 file (checked 1 source file)
```

expected: no errors -- the stub for `multiple` takes `**kwargs: Any`: https://github.com/python/typeshed/blob/508fd844992f5ba8398fec5defa5bb7a44c71269/stdlib/3/unittest/mock.pyi#L91

**Comment by msullivan** [comment]
I think this is intended behavior but also it is unfortunate and frustrating in this case.

It is erroring because patches could contain values for the actual keyword arguments of `mock.patch.multiple` itself. Really it is kind of an API mistake for mock.patch.multiple to take a kwargs and *also* a bunch of keywords arguments...
